src/rips_scan_tag.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+TAG_NAME="$(curl -s https://api.github.com/repos/greenpeace/${RIPS_SCAN_REPO}/tags | jq -r '.[0].name')"
+TAG_COMMIT="$(curl -s https://api.github.com/repos/greenpeace/${RIPS_SCAN_REPO}/tags | jq -r '.[0].commit.sha')"
+
+APPLICATION_ID=${RIPS_APPLICATION_ID}
+
+echo "Downloading zip archive for tag: ${TAG_NAME}"
+
+wget https://github.com/greenpeace/planet4-master-theme/archive/${TAG_COMMIT}.zip
```

Comment:
Hard coded the master-theme repository here isn't very flexible.  Do you want to be able to test against other repos?

I suggest passing two parameters to this file, eg:
```
# Default organisation is 'greenpeace'
org=${1:-greenpeace}
# Default repository is current repo
repo=${2:-$CIRCLE_PROJECT_REPONAME}
```

Reply:
Looking at the output of `org=greenpeace; repo=planet4-master-theme; curl -s https://api.github.com/repos/$org/$repo/tags | jq '.[0]'` it appears we can refer to the `zipball_url` field directly for downloads
```json
{
  "name": "v1.41",
  "zipball_url": "https://api.github.com/repos/greenpeace/planet4-master-theme/zipball/v1.41",
  "tarball_url": "https://api.github.com/repos/greenpeace/planet4-master-theme/tarball/v1.41",
  "commit": {
    "sha": "dc1eecd7296644a91fcb82b74e7e0ac2418292ff",
    "url": "https://api.github.com/repos/greenpeace/planet4-master-theme/commits/dc1eecd7296644a91fcb82b74e7e0ac2418292ff"
  },
  "node_id": "MDM6UmVmODA0MDEyNzk6djEuNDE="
}
```

So to increase script flexibility, might the download instead be
```sh
zipball_url=$(curl -s "https://api.github.com/repos/$org/$repo/tags" | jq '.[0].zipball_url')
wget "$zipball_url" -O "${repo}.zip"

Reply:
That hardcoded line was not intended! I'm using the `RIPS_SCAN_REPO` var to set the repo. Didn't know about `CIRCLE_PROJECT_REPONAME`. I guess we could send 3 parameters to this file using your approach:
* `$1 = org`
* `$2 = repo`
* `$3 = the application ID in rips (currently RIPS_APPLICATION_ID), which is set in the CircleCI config of the master-theme`

Reply:
Is RIPS_APPLICATION_ID a secret at all? If so passing on the command line is no-go.  If it's secret, we can define it in the rips test context ( as per discussion on https://github.com/greenpeace/planet4-master-theme/pull/778 ) and leave it as an env var.  If it's not secret we can still put it in the context, or even hard-code in the file.

Reply:
Not a secret, I defined it in the master theme CircleCI config because it is unique to the master-theme, I think we can add it as third parameter for consistency, as we are using parameters for the other two required vars.

Reply:
Ah I see.  Well, since the first and second parameters are optional, RIPS_APPLICATION_ID should likely be the first parameter.